### DISCUSSION TITLE
Example missing for Torso Decompositions

### DISCUSSION THREAD

**Discussion by vaquent** [esa/SpOC3#9] [Challenge 1: Torso Decompositions]
I think there is no file "UDP.py" for the Challenge1 Torso Decompositions. On the contrary, for the Challenge 2 and 3 we can find them in the directory problems. Can you please put the python file for plotting and try the chromosome example for Torso Decomositions.

**Comment by dietmarwo** [comment]
They are here:

https://api.optimize.esa.int/media/problems/spoc-3-torso-decompositions-small-graph-1711380312223.py
https://api.optimize.esa.int/media/problems/spoc-3-torso-decompositions-medium-graph-1711380328535.py
https://api.optimize.esa.int/media/problems/spoc-3-torso-decompositions-large-graph-1711380341869.py

**Comment by cantordust** [comment]
@vaquent Thank you for your reaching out! 

Please refer to the `_plot()` method in the UDP files (links provided by @dietmarwo above - thank you). The method produces a plot of the Pareto front for the chromosomes provided as input, in terms of fitness based on the two objectives. Please let us know if there is some missing functionality.
